web/www/horas/English/Sancti/12-27.txt

```diff
@@ -91,7 +91,7 @@ R. Who was found worthy that the Lord should honour him more than all the rest,~
 by a special privilege of love.
 
 [Lectio4]
-From the Book on Ecclesiastical writers, written by St. Jerome, Priest (at Bethlehem.) 
```

Comment:
LK used the Bute translation for the English. It includes notes not in the Latin; they're transcribed in the datafiles in brackets and are rendered as rubrics.  This is intentional.  As a rule, please leave these brackets intact.  I'll take this patch as it stands, though.